src/components/Calculator.js

```diff
@@ -1,31 +1,43 @@
-        <button type="button" className="gray-btn">+/-</button>
-        <button type="button" className="gray-btn">%</button>
-        <button type="button" className="orange-btn">/</button>
-        <button type="button" className="gray-btn">7</button>
-        <button type="button" className="gray-btn">8</button>
-        <button type="button" className="gray-btn">9</button>
-        <button type="button" className="orange-btn">*</button>
-        <button type="button" className="gray-btn">4</button>
-        <button type="button" className="gray-btn">5</button>
-        <button type="button" className="gray-btn">6</button>
-        <button type="button" className="orange-btn">-</button>
-        <button type="button" className="gray-btn">1</button>
-        <button type="button" className="gray-btn">2</button>
-        <button type="button" className="gray-btn">3</button>
-        <button type="button" className="orange-btn">+</button>
-        <button type="button" className="gray-btn zero-btn">0</button>
-        <button type="button" className="gray-btn">.</button>
-        <button type="button" className="orange-btn">=</button>
+        <div className="calc-display">{next || total || 0}</div>
+        <button type="button" onClick={this.handleClick} className="gray-btn">AC</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">+/-</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">%</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">÷</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">7</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">8</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">9</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">x</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">4</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">5</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">6</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">-</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">1</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">2</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">3</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">+</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn zero-btn">0</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">.</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">=</button>
```

Comment:
[OPTIONAL] I think you could make this code DRYer, but there's not really a very fancy solution. One thing you could do is abstract all of the specific information of the buttons to an array of objects and then map the array. I will leave it up to you!